src/game/rendering_graph_node.c

```diff
@@ -248,7 +248,7 @@ void switch_ucode(s32 ucode) {
     switch (ucode) {
         default: // GRAPH_NODE_UCODE_DEFAULT
         case GRAPH_NODE_UCODE_DEFAULT:
-            gSPLoadUcodeL(gDisplayListHead++, gspF3DZEX2_PosLight_fifo); // F3DZEX2_PosLight
+            gSPLoadUcodeL(gDisplayListHead++, gspF3DZEX2_NoN_PosLight_fifo); // F3DZEX2_PosLight
```

Comment:
wat

Reply:
That was because of [https://github.com/HackerSM64/HackerSM64/commit/475e039f8acd2d240eab9110597e4d9ecf0609c0](https://github.com/HackerSM64/HackerSM64/commit/475e039f8acd2d240eab9110597e4d9ecf0609c0)